internal/schema/sshKey.go

```diff
@@ -4,9 +4,11 @@ import "github.com/amazeeio/lagoon-cli/pkg/api"
 
 // SSHKey is the basic SSH key information, used by both config and API data.
 type SSHKey struct {
-	Name     string         `json:"name"`
-	KeyValue string         `json:"keyValue"`
-	KeyType  api.SSHKeyType `json:"keyType"`
+	Name           string         `json:"name"`
+	KeyValue       string         `json:"keyValue"`
+	Created        string         `json:"created"`
```

Comment:
I was going to say that the CLI output doesn't have a timezone so maybe you could unmarshal to a `time.Time` here, but looking at the debug output the _response_ doesn't have a timezone either :cry: 

I'll raise a lagoon issue for that.

Reply:
We might be able to just assume UTC in this case even, as I think all API pods run in UTC typically.